pontoon/base/models.py

```diff
@@ -1335,6 +1340,12 @@ class Entity(DirtyFieldsMixin, models.Model):
     comment = models.TextField(blank=True)
     order = models.PositiveIntegerField(default=0)
     source = JSONField(blank=True, default=list)  # List of paths to source code files
+
+    # extra stores data that we want to save for the specific format
+    # this entity is stored in, but that we otherwise don't care
+    # about.
+    extra = JSONField(default=extra_default)
```

Comment:
I'd have to test it to be sure, but probably for the same reason you don't use `{}` as a default function value; every newly-created model instance would share the same dict since default arguments aren't re-evaluated. So if I created a new Translation object, and then modify `extra` without re-assigning it, I'm modifying the dict that all new instances share. Future new instances will have the modifications despite being unrelated to the one I modified.